### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by Dfenestrator**
Design:
1. We create a new doc with an email and signupLink in mongo under the "signups" collection. This is currently a manual process (meaning you have to go into mongo and add this entry), though I plan to add an admin page to make this process simple.
2. We provide any new influencers with the following signup link: /signup/:signupLink
3. If the influencer's email matches with their signup link in the mongo collection, we create a new influencer entry for them. Otherwise we do not sign them up.

There's still some work to be done here - redirecting to error pages, etc. Will try to address with a coupla followup checkins.

### CODE REVIEW

File: app/models/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require('./influencer');
+require('./signup');
```
Comment (tjgupta) [inline app/models/index.js]: I don't see this file in this PR. Did you forget to add it maybe?

Comment (Dfenestrator) [inline app/models/index.js]: Yup, just added it

**Comment by Dfenestrator** [comment]
Added an admin page to generate signup links at /admin. Right now any user is an admin, will address this probably in a followup PR (since the only admin privilege right now is to generate signup links).

File: app/models/index.js (hunk 1 above)
Comment (Dfenestrator) [inline app/models/index.js]: Ugh, I still left out a bunch of files, but should be in the latest commit. Sorry about that

**Comment by tjgupta** [comment]
Looks good to me.

**Pull request merged**